projects/packages/sync/src/class-main.php

```diff
@@ -40,6 +40,30 @@ public static function configure() {
 
 		// Set up package version hook.
 		add_filter( 'jetpack_package_versions', __NAMESPACE__ . '\Package_Version::send_package_version_to_tracker' );
+
+		// Add the custom capabilities for managing modules
+		add_filter( 'map_meta_cap', array( __CLASS__, 'module_custom_caps' ), 10, 2 );
+	}
+
+	/**
+	 * Sets the Module custom capabilities.
+	 *
+	 * @param  string[] $caps Array of the user's capabilities.
+	 * @param  string   $cap  Capability name.
+	 * @return string[] The user's capabilities, adjusted as necessary.
+	 */
+	public static function module_custom_caps( $caps, $cap ) {
```

Comment:
As I've said in the commit message. I'm not certain if this is the best place for it, but it seems the most natural shared package as the Modules class lives in Sync.